### PR TITLE
Style zoom out button

### PR DESCRIPTION
**Pull request by marionb**

### CODE REVIEW

File: thinkhazard/static/less/report.less [hunk 1]
```diff
@@ -149,6 +149,26 @@ body {
   top: 10px;
   left: 10px;
   z-index: 2;
+
+  height: 42px;
+  margin: 5px;
```
Comment (pgiraud) [inline thinkhazard/static/less/report.less]: Margin in the document seem to be actually the position of the element regarding the top left corner of the map.
So we have 2 options here:
 - left: 0; top: 0; margin: 5px;
 - left: 5px; top: 5px; (no margin)

File: thinkhazard/static/less/report.less [hunk 2]
```diff
@@ -149,6 +149,26 @@ body {
   top: 10px;
   left: 10px;
   z-index: 2;
+
+  height: 42px;
```
Comment (pgiraud) [inline thinkhazard/static/less/report.less]: I wouldn't set a height here. Even if specified in the document. The size should automatically be set given the element content by the browser.

File: thinkhazard/static/less/report.less [hunk 3]
```diff
@@ -149,6 +149,26 @@ body {
...
+  margin: 5px;
+  background: rgba(255,255,255, 0.5);
+  border-radius: 5px;
```
Comment (pgiraud) [inline thinkhazard/static/less/report.less]: It looks like all the button have the same border radius.
Let's set it globally by overriding bootstrap's variables.

File: thinkhazard/static/less/report.less [hunk 4]
```diff
@@ -149,6 +149,26 @@ body {
...
+  border: 2px solid rgba(255,255,255,0.75);
+  padding: 5px;
```
Comment (pgiraud) [inline thinkhazard/static/less/report.less]: Let's keep the default value for padding in buttons.
Or, let's change it globally for all the xs buttons (.btn-xs).

File: thinkhazard/static/less/report.less [hunk 5]
```diff
@@ -149,6 +149,26 @@ body {
...
+  font-size: 14px;
```
Comment (pgiraud) [inline thinkhazard/static/less/report.less]: Font-size seem to be the same for all the .btn-xs (overview, download, zoom-out). Let's set the font-size for those buttons globally by overriding bootstrap.

**Comment by marionb** [comment]
Please review

File: thinkhazard/static/less/report.less [hunk 6]
```diff
@@ -137,9 +137,23 @@ body {
...
+
+  &:hover {
+    background: rgba(255,255,255, 0.75);
+  }
+
+  i {
+    background: rgb(255,255,255);
+    color: #58595b;
+    padding: 3px 3.5px;
+    align-items: center;
```
Comment (pgiraud) [inline thinkhazard/static/less/report.less]: Is `align-items` really useful here?
I'd personally avoid too modern css tricks.

File: thinkhazard/static/less/report.less [hunk 7]
```diff
@@ -137,9 +137,23 @@ body {
...
+  border: 2px solid rgba(255,255,255,0.75);
+
+  &:hover {
+    background: rgba(255,255,255, 0.75);
+  }
+
+  i {
+    background: rgb(255,255,255);
+    color: #58595b;
+    padding: 3px 3.5px;
```
Comment (pgiraud) [inline thinkhazard/static/less/report.less]: "3.5px"? really?

**Comment by pgiraud** [comment]
Please merge.

**Pull request merged**